addons/payment/views/portal_templates.xml

```diff
@@ -142,18 +142,109 @@
                 <div class="container">
                     <!-- Portal breadcrumb -->
                     <t t-call="payment.portal_breadcrumb"/>
-                    <!-- Status page -->
+                    <!-- Payment status page -->
                     <div name="o_payment_status" class="row justify-content-center my-3">
-                        <div name="o_payment_status_content"
-                             class="col-12 col-lg-8">
-                            <!-- The content is generated in JavaScript -->
+                        <div name="o_payment_status_content" class="col-12 col-lg-8">
+                            <t t-call="payment.transaction_details"/>
                         </div>
                     </div>
                 </div>
             </div>
         </t>
     </template>
 
+    <template id="payment.payment_not_found" name="Payment Not Found">
```

Comment:
The wrappers around the template contents are indeed identical (layout, title, container, breadcrumbs), but the contents themselves are quite different. One shows a simple error message, the other loads a status page via `t-call`.

They could be merged, since they're rendered in the same python method, but I'm not sure there's a real benefit in doing so. WDYT?

Reply:
Yes, that is why i thought it could be a good idea to `t-call` the status page `t-if="tx_found"` for example, and show an error message `t-else`. That way it is easier to modify the view around (layout, title, container, breadcrumbs) only one time instead of making changes in both templates. However, if you find 2 templates are indeed needed, you can leave it like that, it is not blocking ofc.

Reply:
I'm fine with either so I updated it :)